docs/src/main/asciidoc/spring-integration-pubsub.adoc

```diff
@@ -43,6 +43,8 @@ The `PubSubInboundChannelAdapter` delegates the conversion to the desired payloa
 
 To use the inbound channel adapter, a `PubSubInboundChannelAdapter` must be provided and configured on the user application side.
 
+NOTE: The subscription name could either be a short subscription name within the current project, or the fully-qualified name referring to a subscription in a different project using the `projects/[project_name]/subscriptions/[subscription_name]` format.
+
 [source,java]
```

Comment:
Do you mind also adding these updates to `src/main/md/spring-integration.md`, so we don't need to regenerate the md files?

Reply:
Done; changed the format a bit.